scripts/apps-version-check.exs

```diff
@@ -0,0 +1,247 @@
+#!/usr/bin/env elixir
+
```

Comment:
add some doc to describe the purpose and usage of this script

Reply:
:checkered_flag: